bip-musig2.mediawiki

```diff
@@ -145,13 +148,17 @@ However, if ''PartialSigVerify'' succeeds for all partial signatures then ''Part
 
 ==== Tweaking ====
 
+The aggregate public key can be ''tweaked'', which modifies the key as defined in the [[#tweaking-definition|Tweaking Definition]] subsection.
+In order to apply a tweak, the KeyGen Context output by ''KeyAgg'' is provided to the ''ApplyTweak'' algorithm with the ''is_xonly_t'' argument set to false for ordinary tweaking and true for X-only tweaking.
+The resulting KeyGen Context can be used to apply another tweak with ''ApplyTweak'' or obtain the aggregate public key with ''GetPubkey''.
+
 In addition to public keys, the ''KeyAgg'' algorithm accepts tweaks, which modify the aggregate public key as defined in the [[#tweaking-definition|Tweaking Definition]] subsection.
 For example, if ''KeyAgg'' is run with ''v = 2'', ''is_xonly_t<sub>1</sub> = false'', ''is_xonly_t<sub>2</sub> = true'', then the aggregate key is first ordinarily tweaked with ''tweak<sub>1</sub>'' and then X-only tweaked with ''tweak<sub>2</sub>''.
 
 The purpose of specifying tweaking is to ensure compatibility with existing uses of tweaking, i.e., that the result of signing is a valid signature for the tweaked public key.
 The MuSig2 algorithms take arbitrary tweaks as input but accepting arbitrary tweaks may negatively affect the protocol's security.
 Instead, signers should obtain the tweaks according to other specifications.
-This typically involves deriving the tweaks from a hash of the aggregate public key and some other information.
+This typically involves deriving the tweaks from a hash of the aggregate public key (obtained with ''GetPubkey'' or ''GetPubkey33'') and some other information.
```

Comment:
... here I was surprised that it appears of the blue.

Reply:
Added a few sentences to explain `GetPubkey33`